adapters/repos/db/migrator.go

```diff
@@ -128,7 +128,14 @@ func (m *Migrator) AddClass(ctx context.Context, class *models.Class,
 }
 
 func (m *Migrator) DropClass(ctx context.Context, className string) error {
-	return m.db.DeleteIndex(schema.ClassName(className))
+	if err := m.db.DeleteIndex(schema.ClassName(className)); err != nil {
+		return err
+	}
+
+	if m.cloud != nil {
```

Comment:
If we error deleting the index in the DB (for whichever reason) do we want to *not* delete the cloud shard(s) ? Or should we ignore the error above and still complete the deletion of the cloud shard as well ?

Reply:
i'd say if it wasn't deleted from the db it shouldn't be deleted on the cloud